xml/Microsoft.Azure.Documents.Client/RequestOptions.xml

```diff
@@ -903,7 +903,7 @@
              The token for use with session consistency.
              </value>
         <remarks>
-             One of the <see cref="P:Microsoft.Azure.Documents.Client.RequestOptions.ConsistencyLevel" /> for Azure Cosmos DB is Session. In fact, this is the deault level applied to accounts.
+             One of the <see cref="P:Microsoft.Azure.Documents.Client.RequestOptions.ConsistencyLevel" /> for Azure Cosmos DB is Session. In fact, this is the default level applied to accounts.
```

Comment:
This file is generated based on source code and then automatically sync'd to this repository.  Any changes made here will be overwritten the next time a sync is performed.  

The `Microsoft.Azure.DocumentDB` package associated with this file is not open-source.  To update it, you'll need to coordinate with the Cosmos team.   

Because the package is deprecated, they may no longer be pushing new builds and may ask that this gets updated here...  so I'll leave this open for now until we have an answer.

Reply:
Thank you @jsquire, we can safely reject this PR.